DanielVargas/tic_tac_toe/src/com/company/TicTacToe.java

```diff
@@ -0,0 +1,154 @@
+package com.company;
+
+import java.util.Scanner;
+
+public class TicTacToe {
+    final int cols = 3;
+    final int rows = 3;
+    private String[][] board;
+    private String player1;
+    private String player2;
+    private int num_of_plays;
+
+    TicTacToe(String player1, String player2) {
+        this.board = new String[this.cols][this.rows];
+        this.player1 = player1;
+        this.player2 = player2;
+        this.num_of_plays = 0;
+        for (int i = 0; i < this.rows; i++) {
+            for (int j = 0; j < this.cols; j++) {
+                this.board[i][j] = "";
+            }
+        }
+    }
+
+    public void printBoard() {
+        System.out.println("+--+--+--+");
+        for (int i = 0; i < this.rows; i++) {
+            System.out.print("| ");
+            for (int j = 0; j < this.cols; j++) {
+                if (this.board[i][j].length() > 0)
+                    System.out.print(this.board[i][j]);
+                else
+                    System.out.print(" ");
+                System.out.print("| ");
+            }
+            System.out.println("");
+            System.out.println("+--+--+--+");
+        }
+    }
+
+    private boolean setPlay(int row, int col, String player) {
+        String mark = "X";
+        if (player.equals (this.player2)) {
+            mark = "O";
+        }
+        if (check_position(row, col)) {
+            this.board[row][col] = mark;
+            this.num_of_plays++;
+            this.printBoard();
+            return true;
+        } else {
+            return false;
+        }
+
+    }
+
+    private boolean check_position(int row, int col) {
```

Comment:
This method name does not follow java conventions, **please update all methods and variables names according to this:**

_Please follow java convention to declare variable/method names, when the name is compound for many words the first letter of the word is in lowercase the remaining ones are written with uppercase:_

```java
int freePositionsOnBoard;

public void printBoard() {
}
```

Reply:
@DannyVargas are you going to solve this???